near/omni-types/src/evm/header.rs

```diff
@@ -0,0 +1,159 @@
+pub use ethereum_types::{Address, Bloom, H256, H64, U256, U64};
+use rlp::{Decodable, DecoderError, Encodable, Rlp, RlpStream};
+
+use super::utils::keccak256;
+
+#[derive(Default, Debug, Clone)]
```

Comment:
BorshSerialize, BorshDeserialize?

Reply:
Unfortunately, the types don't support borsh, and I don't want to move all the `eth-types`, also, as mentioned by @kiseln, we are going to replace the types to alloy at some point.